### PR TITLE
Cb ghpages branch

### PR DESCRIPTION
**Pull request by CaitlinBate**
deploy notebook done, small edited to project and tag notebooks

### CODE REVIEW

File: _episodes/07_deploy_metadata.md [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+
+If new stations are identified:
+- Compare these station names to existing stations in the `stations` table in the database. Are they truly new, or is there a typo in the raw table? Did the station change names, but is in the same location as a previous station?
+- If there are fixes to be made, change the records in the `raw` table, or contact the researcher.
+- If all stations are truly new, you should review the information in the editable form, then select `Add New Stations`.
+
+The success message will look like:
+
+```
+TODO ---- whats the success message?
```
Comment (naomitress) [inline _episodes/07_deploy_metadata.md:225]: need to run some records to find this

**Review by naomitress — APPROVED** [review]

File: _episodes/07_deploy_metadata.md (hunk 1 above)
Comment (naomitress) [reply _episodes/07_deploy_metadata.md:225]: carry this todo over to the gh-pages branch

**Review by PatrickWMartin — APPROVED** [review]
Removed tag todos and think its good to go

**Review by jdpye — APPROVED** [review]
Tidied up a typo or two and harmonized the spelling of GitLab with the other PR that's pending. Looking good

**Pull request merged**